### PR TITLE
add version number to user agent header

### PR DESCRIPTION
**Pull request by morningvera**
Couldn't think of an ideal way to automate this so I ended up writing a script. Currently, only admins can run it.

### CODE REVIEW

**Review by jbonzo — APPROVED** [review]

File: rest/client/client.go [hunk 1]
```diff
@@ -29,7 +31,7 @@ func New(apiKey string) Client {
 	c.SetAuthToken(apiKey)
 	c.SetRetryCount(DefaultRetryCount)
 	c.SetTimeout(10 * time.Second)
-	c.SetHeader("User-Agent", "Go client")
+	c.SetHeader("User-Agent", fmt.Sprintf("Go client %v", clientVersion))
```
Comment (jbonzo) [inline rest/client/client.go:34]: before major release we will want a better user agent that includes `Polygon` or something. And have it be standard across the other client libs

Comment (morningvera) [reply rest/client/client.go:34]: agreed, maybe we should change that now? `Polygon Go client v0.7.1`?

Comment (jbonzo) [reply rest/client/client.go:34]: `Polygon.io GoClient/v0.7.1`
or 
`Polygon.io GoClient/0.7.1`

Comment (morningvera) [reply rest/client/client.go:34]: i think i'll go with the first one so it matches the repo tags

**Review by jbonzo — APPROVED** [review]

**Pull request merged**